README.md

```diff
@@ -44,16 +45,10 @@ See [INSTALL.md](/INSTALL.md) for detailed installation instructions.
 
 ### Versioning
 
-`client-go` follows [semver](http://semver.org/). We will not make
-backwards-incompatible changes without incrementing the major version number. A
-change is backwards-incompatible either if it *i)* changes the public interfaces
-of `client-go`, or *ii)* makes `client-go` incompatible with otherwise supported
-versions of Kubernetes clusters.
+- For each `v1.x.y` Kubernetes release, the major version (first digit)
+would remain `0`.
 
-Changes that add features in a backwards-compatible way will result in bumping
-the minor version (second digit) number.
-
-Bugfixes will result in the patch version (third digit) changing. PRs that are
+- Bugfixes will result in the patch version (third digit) changing. PRs that are
 cherry-picked into an older Kubernetes release branch will result in an update
 to the corresponding branch in `client-go`, with a corresponding new tag
 changing the patch version.
```

Comment:
just noticed the text below:
> A consequence of this is that `client-go` version numbers will be unrelated to
> Kubernetes version numbers.

That's not quite true now... they are related, as described above. I would drop that sentence as well.

Reply:
Done